src/compose/getComposeInputPlaceholder.js

```diff
@@ -0,0 +1,23 @@
+import {
+  isStreamNarrow,
+  isTopicNarrow,
+  isPrivateNarrow,
+  isGroupNarrow,
+} from '../utils/narrow';
+
+export default (narrow, auth, users) => {
+  if (isGroupNarrow(narrow)) {
+    return 'Message group';
+  } else if (isPrivateNarrow(narrow)) {
+    if (auth && narrow[0].operand === auth.email) return 'Jot down something';
+    else {
+      const user = users.find(u => u.email === narrow[0].operand);
+      return user && `Message @${user.fullName}`;
+    }
+  } else if (isStreamNarrow(narrow)) {
+    return `Message #${narrow[0].operand}`;
+  } else if (isTopicNarrow(narrow)) {
+    return `Message #${narrow[0].operand} topic:${narrow[1].operand}`;
+  }
+  return 'Type a message';
+};
```

Comment:
Simplify the function by not having the 'else' since 'return' already halts the execution of the function.